src/main.cpp

```diff
@@ -782,6 +782,7 @@ int main(int Argc, char *Argv[]) {
             if (2 <= val && val <= 4) {
```

Comment:
The latest DWARF version is 5. I'm not sure if LLVM supports it (and what version of LLVM supports it). I suggest checking it and if it's supported changing the range of supported versions.

Reply:
It looks like LLVM started to somehow support DWARF v5 since 7 ([link](https://releases.llvm.org/7.0.0/docs/ReleaseNotes.html)). I have added 5 as option as the separate commit.